src/dispatch.rs

```diff
@@ -68,15 +68,16 @@ impl AsyncDispatcher {
     }
 
     /// Waits for all the systems to finish
```

Comment:
Should probably have documentation stating that it's not in fact waiting for "all the systems to finish".